### DISCUSSION TITLE
Pertanyaan tentang Consumer ID atau X-cons-id

### DISCUSSION THREAD

**Discussion by kangketikonlen** [virusphp/service-bridging#9] [Q&A]
Halo, saya lagi belajar bridging bpjs kesehatan. Mau tanya, kalo ConsumerID atau X-cons-id itu dapet darimana ya?

Makasih sebelumnya :)

**Comment by virusphp** [comment]
Halo terimakasih sudah mampi,

untuk mendapatkan CostumerID dan X-Const-id adalah dengan  melakukan MOU dengan BPJS kesehatan

biasanya yang dapat melakukan MOU adalah RUMAH SAKIT maupun Klinik kesehatan serta Vendor yang sudah mempunyai Client Klinik kesehatan 

terimaksih
